src/ProgramBlockEditor.test.js

```diff
@@ -942,20 +928,3 @@ describe('Using change character position by column/row labels', () => {
         expect(characterYPositionCoordinateBox.props.onChange).toBe(undefined);
     });
 });
-
-describe('Character position gets updated on character-position div', () => {
```

Comment:
If this is still worth testing (and it seems like it is), it should be moved to `App.test.js` instead of removing it.

Reply:
Yeah, I agree that the test is worth testing; however, we have only used App test file to see if the coding environment has rendered, so we haven't been writing test cases for other logics in App. I see that test cases for AudioImpl has added to it, and was wondering if we should write more test cases for App? What do you think @sbates-idrc?